packages/pigment-css-react/src/createExtendSxProp.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Runtime function for creating `extendSxProp`.
+ *
+ * This is a placeholder for the runtime function for replacing the `extendSxProp` in Material UI
+ * to reduce bundle size only. DO NOT use it in your codebase.
+ */
+function extendSxProp(props) {
+  return props;
+}
+export default function createExtendSxProp() {
```

Comment:
Let's also mark it as `@internal` for later documentation

Reply:
Can you also add a fixture test for it ?

Reply:
Sure, I will.

Reply:
To be safe, should we rename it to: `internal_createExtendSxProp`, maybe even in @mui/material we can keep this name.

Reply:
Done!